components/link-social/src/tokens.json

```
@@ -6,73 +6,103 @@
+        "type": "color"
       },
-      "border-width": {
+      "border-color": {
         "$extensions": {
           "nl.nldesignsystem.css.property": {
-            "syntax": "<length>",
+            "syntax": "<color>",
             "inherits": true
-          }
-        }
+          },
+          "nl.nldesignsystem.figma.supports-token": false
+        },
+        "type": "color"
       },
-      "border-color": {
+      "border-width": {
         "$extensions": {
           "nl.nldesignsystem.css.property": {
-            "syntax": "<color>",
+            "syntax": "<length>",
             "inherits": true
-          }
-        }
+          },
+          "nl.nldesignsystem.figma.supports-token": false
+        },
+        "type": "borderWidth"
       },
-      "size": {
+      "color": {
         "$extensions": {
           "nl.nldesignsystem.css.property": {
-            "syntax": "<length>",
+            "syntax": "<color>",
             "inherits": true
-          }
-        }
+          },
+          "nl.nldesignsystem.figma.supports-token": false
+        },
+        "type": "color"
       },
       "margin-inline-start": {
         "$extensions": {
           "nl.nldesignsystem.css.property": {
             "syntax": "<length>",
             "inherits": true
-          }
-        }
+          },
+          "nl.nldesignsystem.figma.supports-token": false
+        },
+        "type": "spacing"
       },
-      "color": {
+      "size": {
         "$extensions": {
           "nl.nldesignsystem.css.property": {
-            "syntax": "<color>",
+            "syntax": "<length>",
             "inherits": true
-          }
-        }
+          },
+          "nl.nldesignsystem.figma.supports-token": false
+        },
+        "type": "sizing"
       },
-      "hover": {
-        "scale": {
+      "icon": {
+        "size": {
           "$extensions": {
             "nl.nldesignsystem.css.property": {
               "syntax": "<length>",
               "inherits": true
-            }
-          }
-        },
+            },
+            "nl.nldesignsystem.figma.supports-token": false
+          },
+          "type": "sizing"
+        }
+      },
+      "hover": {
         "background-color": {
           "$extensions": {
             "nl.nldesignsystem.css.property": {
               "syntax": "<color>",
               "inherits": true
-            }
-          }
+            },
+            "nl.nldesignsystem.figma.supports-token": false
+          },
+          "type": "color"
         },
         "color": {
           "$extensions": {
             "nl.nldesignsystem.css.property": {
               "syntax": "<color>",
               "inherits": true
-            }
-          }
+            },
+            "nl.nldesignsystem.figma.supports-token": false
+          },
+          "type": "color"
+        },
+        "scale": {
+          "$extensions": {
+            "nl.nldesignsystem.css.property": {
+              "syntax": "<length>",
```

Deze property is een soort zoom-level, zonder CSS unit.

```suggestion
              "syntax": "<number>",
```